bridge/qb/client/functions.lua

```diff
@@ -153,17 +266,27 @@ function functions.SpawnVehicle(model, cb, coords, isnetworked, teleportInto)
     if cb then cb(veh) end
 end
 
----@deprecated use DeleteVehicle from imports/utils.lua
-functions.DeleteVehicle = DeleteVehicle
+---@deprecated use qbx.deleteVehicle from modules/lib.lua
+functions.DeleteVehicle = qbx.deleteVehicle
 
----@deprecated use GetPlate from imports/utils.lua
-functions.GetPlate = GetPlate
+---@deprecated use qbx.getVehiclePlate from modules/lib.lua
+functions.GetPlate = function(vehicle)
+    if vehicle == 0 then return end
+    return qbx.getVehiclePlate(vehicle)
+end
 
----@deprecated use GetVehicleDisplayName from imports/utils.lua
-functions.GetVehicleLabel = GetVehicleDisplayName
+---@deprecated use qbx.getVehicleDisplayName from modules/lib.lua
+functions.GetVehicleLabel = function(vehicle)
+    if vehicle == nil or vehicle == 0 then return end
+    return qbx.getVehicleDisplayName(vehicle)
+end
 
----@deprecated use IsVehicleSpawnClear from imports/utils.lua
-functions.SpawnClear = IsVehicleSpawnClear
+---@deprecated use lib.getNearbyVehicles from ox_lib
+functions.SpawnClear = function(coords, radius)
+    local vehicles = lib.getNearbyVehicles(coords, radius)
+    if #vehicles > 0 then return false end
+    return true
```

Comment:
```suggestion
    return #vehicles == 0
```
This can be simplified.